doc/man3/OSSL_HPKE_CTX_new.pod

```diff
@@ -455,6 +467,16 @@ codepoints. Valid (case-insensitive) names are:
 String variants of the numbers listed in L</OSSL_HPKE_SUITE Identifiers>
 can also be used.
 
+=head2 High-level functions
+
+OSSL_HPKE_BOX_seal() calls OSSL_HPKE_encap and OSSL_HPKE_seal, and serializes the
+outputs into a newly allocated buffer, whose address is returned in box and sets boxlen
+accordingly. It is the caller's responsibility to free I<*box>.
+
+OSSL_HPKE_BOX_open() calls OSSL_HPKE_decap and OSSL_HPKE_open, and returns a
+newly allocated copy of the plaintext in ptx, and sets ptlen to the length. It is
+the caller's responsibility to free I<*ptx>.
```

Comment:
The problem I see with this is that this invents a new format that is undocumented and non-standard. I am slightly worried of the possible consequences. Will we have to support it forever, is it extensible enough, etc.

Reply:
Maybe we do not need the format to be formally standardized but at least a precise description of it should be included in this PR so independent implementations are possible.

Reply:
I'm wary of introducing something non-standard. Maybe an OTC discussion is in order. In any case I'd suggest we include some kind of "version" specifier in the format so that it would be easy to make changes to the format in the future if we deem it necessary.

Reply:
I was thinking the same thing in regards to a version